tests/servers/tests.py

```diff
@@ -174,3 +175,23 @@ def test_database_writes(self):
             ['jane', 'robert', 'emily'],
             lambda b: b.name
         )
+
+
+class LiveServerPort(LiveServerBase):
+
+    def test_port_bind(self):
+        """
+        Ensure that each LiveServerTestCase binds to a unique port
```

Comment:
You can break the docstring close to 79 character and use this style for the ticket reference:
```
        """
        Ensure that each LiveServerTestCase binds to a unique port or fails to
        start a server thread when run concurrently (#26011).
        """
```